ui/shared/components/form/ReduxFormWrapper.jsx

```diff
@@ -268,7 +271,11 @@ class ReduxFormWrapper extends React.Component {
   }
 
   componentWillUpdate(nextProps) {
-    if (nextProps.submitSucceeded && nextProps.closeOnSuccess && !nextProps.noModal) {
+    if (this.props.onSubmitSucceeded && nextProps.submitSucceeded) {
+      this.props.onSubmitSucceeded()
+      this.props.handleClose(true)
```

Comment:
I don't think this should call `handleClose` here. I think you should make this its own `if` statement separate from the `if/else` block that already existed so that way it does 2 separate checks, one for if it should call `onSubmitSucceeded` and the other for if it should call `handleClose`